src/components/input/input.html

```diff
@@ -0,0 +1,108 @@
+ <a href="../../../theme.html"> <li>Theme</li></a>
+
+<li class="list-head">UI Components</li>      
+ <a href="../alert/alert.html"> <li>Alert</li></a>
+
+<a href="../avatar/avatar.html"><li> Avatar</li></a>
+ <a href="../badge/badge.html"> <li>Badge</li></a>
+ <a href="../button/button.html"> <li>Button</li></a>
+ <a href="../card/card.html"> <li>Card</li></a>
+ <a href="../grid/grid.html"> <li>Grid</li></a>
+ <a href="../image/image.html"> <li>Image</li></a>
+ <a href="../input/input.html"> <li class="active">Input</li></a>
+ <a href="../lists/lists.html"> <li>Lists</li></a>
+ <a href="../modal/modal.html"> <li>Modal</li></a>
+ <a href="../navigation/navigation.html"><li>Navigation</li></a>
+ <a href="../rating/rating.html"> <li>Rating</li></a>
+ <a href="../toast/toast.html"> <li>Toast</li></a>
+</div>
+</aside>
+<main class="doc-main"> 
+    <nav>
+        <ul class="nav-group start">
+          <li> <a href="../../../docs.html">Getting Started</a></li>
+            <li class="nav-active"><a href="../avatar/avatar.html">UI Components</a></li>
+        </ul>
+        <ul class="nav-group end">
+        <li><a href="https://github.com/shangkaul/krozaro_component_lib"><i class="icon icon-md fa fa-github"></i></a></li>
+        <li></li>
+    </ul>
+    
+    </nav>
+    <div class="doc-content">
+        <section class="component">
+            <h1>Input</h1>
+          <p class="md"> The input element is a modification on the standard HTML imput element. 
+            It used to accept data from the user. The data can be of type email, text, number,tel,url etc.
+          </p>  
+          </section>
+
+          <section class="component">
+            <section class="demo">
+              <div class="flex-col">
+              <input class="textbox" placeholder="Simple Input"/>
+              <input class="textbox textbox-success" placeholder="Success"/>
+              <input class="textbox textbox-fail" placeholder="Fail"/>
+
+              </div>
+            </section>
+          </section>
+<pre>
+  <code class="language-html">
+&lt;input class=&quot;textbox&quot; placeholder=&quot;Simple Input&quot;/&gt;
+&lt;input class=&quot;textbox textbox-success&quot; placeholder=&quot;Success&quot;/&gt;
+&lt;input class=&quot;textbox textbox-fail&quot; placeholder=&quot;Fail&quot;/&gt;  </code>
+</pre>
+           <footer>
+            <a class="prev" href="../image/image.html"> Image </a>
+            <a class="next" href="../list/list.html"> List </a>
+          </footer>
+</div>
+</main>
+<script src="../../../prism/prism.js"></script>
+  </body>
+  
+</html>
+
+
+
+<!-- <!DOCTYPE html>
+<html>
+  <head>
+    <title>Krozaro</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="../../krozaro.styles.min.css" />
+
+  </head>
+  <body>
+    <div class="container">
+      <input class="textbox textbox-success" />
+      <input class="textbox textbox-fail" />
+      <input class="textbox" />
+    </div>
+  </body>
+</html> -->
```

Comment:
remove comments, Otherwise looks good to me for merging

Reply:
Thanks, will remove this.